core/app/models/mno_enterprise/kpi.rb

```diff
@@ -0,0 +1,6 @@
+module MnoEnterprise
+  class Kpi < BaseResource
```

Comment:
No indeed, I don't see any advantage of the namespacing compared to the pain and suffering it is bringing, by usually not working, because it gets off the beaten track.